### PR TITLE
Added new types for html5plus

### PR DESCRIPTION
**Pull request by zhetengbiji**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/html5plus/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```
Comment (sandersn) [inline types/html5plus/tslint.json]: Please try to make new types lint cleanly, without lint rule overrides.

File: types/html5plus/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,15476 @@
+// Type definitions for html5plus 1.0.0
```
Comment (sandersn) [inline types/html5plus/index.d.ts]: is this the project's name correct? I can only find html5-plus on npm.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/html5plus/index.d.ts (hunk 2 above)
Comment (zhetengbiji) [reply types/html5plus/index.d.ts]: Yes, the project name is correct, but this project is not on the NPM.

Comment (sandersn) [reply types/html5plus/index.d.ts]: I am worried about name clashes with future npm projects. Do you think html5plus is likely to be added to npm in the future? Or do you think a similar, future project might want to use the html5plus name on npm?

Comment (zhetengbiji) [reply types/html5plus/index.d.ts]: This name does not conflict with future NPM projects. html5plus is the product name of dcloud.io. If a future project appears on NPM under the name html5plus, it refers to the current project. For example, the html5-plus project on NPM is a d.ts file written by a third party for this project.

**Review by sandersn — APPROVED** [review]

**Pull request merged**